.github/container/Dockerfile.pax.arm64

```diff
@@ -1,65 +1,143 @@
 # syntax=docker/dockerfile:1-labs
+
+ARG BASE_IMAGE=ghcr.io/nvidia/jax:mealkit
+ARG SRC_PATH_PAXML=/opt/paxml
+ARG SRC_PATH_PRAXIS=/opt/praxis
+ARG SRC_PATH_TFTEXT=/opt/tensorflow-text
+ARG SRC_PATH_LINGVO=/opt/lingvo
+
+###############################################################################
+## build tensorflow-text and lingvo, which do not have working arm64 pip wheels
+###############################################################################
+
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE} as wheel-builder
+
+# tensorflow-text and lingvo build needs bazel
+RUN wget https://github.com/bazelbuild/bazelisk/releases/download/v1.17.0/bazelisk-linux-arm64 -O /usr/bin/bazel && \
+    chmod a+x /usr/bin/bazel
+
+#------------------------------------------------------------------------------
+# build tensorflow-text 2.13.0 from source
+#------------------------------------------------------------------------------
+
+FROM wheel-builder as tftext-builder
+ARG SRC_PATH_TFTEXT
+RUN <<"EOT" bash -exu -o pipefail
+pip install tensorflow_datasets==4.9.2 auditwheel tensorflow==2.13.0
+get-source.sh -l tensorflow-text -m ${MANIFEST_FILE}
+cd ${SRC_PATH_TFTEXT}
+./oss_scripts/run_build.sh
+EOT
+
+#------------------------------------------------------------------------------
+# build lingvo
+#------------------------------------------------------------------------------
+
+FROM wheel-builder as lingvo-builder
+ARG SRC_PATH_TFTEXT
+ARG SRC_PATH_LINGVO
+
+COPY --from=tftext-builder ${SRC_PATH_TFTEXT}/tensorflow_text*.whl /opt/
+
+RUN get-source.sh -l lingvo -m ${MANIFEST_FILE}
+
+# build lingvo
+RUN <<"EOT" bash -exu -o pipefail
+
+pushd ${SRC_PATH_LINGVO}
+git fetch origin pull/329/head:pr329
+git cherry-pick --allow-empty pr329
+
+# Disable 2 flaky tests here
```

Comment:
Why is it here but not a part of manifest file?

Reply:
I did not add the patch for lingvo, but I'm all for moving the patch into the manifest. Would it be okay to address this in a follow up PR? If so, I can file an issue

Reply:
TODO: Create issue

Reply:
https://github.com/NVIDIA/JAX-Toolbox/issues/409